helm/dagger/values.yaml

```diff
@@ -150,14 +139,29 @@ engine:
   hostPath:
     # Use hostPath for data volume
     dataVolume:
-      # When disabled, PVC will be used if configured above
+      # When disabled, PVC will be used if configured below
       enabled: true
 
     # Use hostPath for run volume
     runVolume:
       # When disabled, no run volume will be mounted
       enabled: true
 
+  ### Persistent Volume Claim configuration
+  #
+  persistentVolumeClaim:
+    # Use PersistentVolumeClaim for data volume
+    enabled: false
+
+    # PVC specification
+    volumeClaimTemplate:
```

Comment:
Do we need this extra layer of indirection? We don't use this `template` kind of style anywhere else.